pkg/grub/rootfs.cfg

```diff
@@ -343,15 +343,7 @@ function set_xen_boot {
 }
 
 function set_eve_flavor {
-   # if eve_flavor is not set explicitly we assume it is
-   # embedded in the end of the name of the image
-   if [ "$eve_flavor" != xen -a "$eve_flavor" != kvm ] ; then
-      if regexp -- '-xen(-amd64$|-arm64$|-riscv64$|$)' $rootfs_title ; then
-         eve_flavor=xen
-      else
-         eve_flavor=kvm
-      fi
-   fi
+   cat -s eve_flavor /etc/eve-hv-type
```

Comment:
I don't know whether the build will produce a file with a CR at the end or not, but is this cat -s robust and can handle with and without?